src/app/ApplicationTemplate.Mobile/ApplicationTemplate.Mobile.csproj

```diff
@@ -11,6 +11,16 @@
 		<SupportedOSPlatformVersion Condition="'$(TargetFramework)' == 'net6.0-android'">26.0</SupportedOSPlatformVersion>
 		<SupportedOSPlatformVersion Condition="'$(TargetFramework)' == 'net6.0-ios'">14.2</SupportedOSPlatformVersion>
 	</PropertyGroup>
+	<ItemGroup>
+	  <None Remove="Android\Assets\Fonts\Guillon-Black.ttf" />
```

Comment:
Are they actually any style that uses those fonts?

Reply:
Don't know, will have to check, I just added what was in the old template. I can also provide screenshots if it does. Also I see that the file as been updated, I will need to check that out.

Reply:
Feel like it's working on Windows. I have to check mobile.

Before:
![image](https://user-images.githubusercontent.com/57812285/215855891-accec1a9-e4e8-4d72-83c3-5927fe2b8d0a.png)

After:
![image](https://user-images.githubusercontent.com/57812285/215855921-69741fe1-f0b2-4e2b-a5af-3d3789b9bb9d.png)